Replicator/tests/ReplicatorCollectionSGTest.cc

```diff
@@ -915,7 +915,7 @@ TEST_CASE_METHOD(ReplicatorCollectionSGTest, "Auto Purge Enabled - w/ and w/o Fi
             pullFilterPurge = 0;
         }
 
-        unsigned collIndex;
+        unsigned collIndex = -1;
```

Comment:
Kind of interesting that this wouldn't set of some kind of signed/unsigned mismatch...